test/utils/common-utils.test.ts

```diff
@@ -16,7 +18,7 @@ describe("Utils - Common Utils", () => {
       expect(actual).toBe(min);
     });
 
-    it.each([-3, 0, 3])("should not alter values between min and max (%i)", (value) => {
+    it.each(range)("should not alter values between min and max (%i)", (value) => {
```

Comment:
I just chose 1 negative neutral, 0 and one positive neutral. But I guess this works too as these tests don't take much time and resources